cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,59 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
-
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import UserInformationPageObject 
+  from '../support/pages/userInformation.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+
+const signInPage = new SignInPageObject();
+const userInformation = new UserInformationPageObject();
+const homePage = new HomePageObject();
 describe('Settings page', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
+    cy.task('db:clear');
+    
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+    homePage.clickOnSettings();
 
   });
 
   it('should provide an ability to update username', () => {
-
+    userInformation.usernameField.clear();
+    userInformation.typeUsername(user.username1);
+    userInformation.clickOnUpdateSettingsBtn();
+    homePage.clickOnSettings();
+    userInformation.usernameField.should('have.value', user.username1);
```

Comment:
```suggestion
    userInformation.usernameField.should('have.value', user.username1);
```
wrap this and similar methods into the page object method